R/readWorkbook.R

```diff
@@ -137,16 +143,19 @@ wb_read <- function(
   )
 
 }
-
-#' @name read_sheet_names
-#' @title Get names of worksheets
-#' @description Returns the worksheet names within an xlsx file
+# `read_sheet_names()` ----------------------------------------------
+#' Get names of worksheets
+#'
+#' Returns the worksheet names within an xlsx file
+#'
 #' @param file An xlsx or xlsm file.
 #' @return Character vector of worksheet names.
 #' @examples
 #' wb_load(system.file("extdata", "openxlsx2_example.xlsx", package = "openxlsx2"))$get_sheet_names()
+#' @keywords internal
```

Comment:
Put `@keywords internal` as it is deprecated. It won't show up in the function reference, but if a user types `?read_sheet_names`, they will be able to access the help topic,

Reply:
Now I'm confused, why did you remove all the other keywords internal roxygen comments?

Reply:
Okay, thanks for the explanation. I might have picked it up somewhere on stack overflow